_posts/2015-08-10-type-driven-development.html

```diff
@@ -74,7 +74,7 @@
     When in the <em>Ready</em> or <em>No message</em> states, the Polling Consumer may also realise that time is running out, and so decide to quit, in which case the (final) state is the <em>End</em> state.
   </p>
   <p>
-    This means that the Polling Consumer needs to measure what it does, and those measurements influence what it decides to do.
+    This means that the Polling Consumer needs to measure what it does, and have those measurements influence what it decides to do.
```

Comment:
I could also have written the original sentence:

*This means that the Polling Consumer needs to measure what it does; those measurements influence what it decides to do.*

I think that introducing "have" changes the meaning of the sentence ever so slightly. Not that it matters much. Do you think your proposed change makes it *easier* to understand the sentence?